### PR TITLE
Feature/stateless functions

### PR DESCRIPTION
**Pull request by MontJamesmont**

### CODE REVIEW

File: src/cv/firstPage/firstPage.js [hunk 1]
```diff
@@ -13,79 +13,77 @@ import experience from '../../img/experience.svg';
...
-            education: education
-        }
+function FirstPage(props) {
+    let title = "";
+    let headerLogo = "";
+    let user = props.user;
+    let sources = props.sources;
+    let experiencesList = [];
+    let educationList = [];
+    let dafaultSources = {
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: defaultSources

File: src/cv/firstPage/firstPage.js [hunk 2]
```diff
@@ -13,79 +13,77 @@ import experience from '../../img/experience.svg';
...
-                dafaultSources.headerLogo = test;
-            } else if (title.includes('bussiness')) {
-                dafaultSources.headerLogo = bussinessAnalyst;
-            } else if (title.includes('cq5')) {
-                dafaultSources.headerLogo = cq5;
-            }
+    if (user.title) {
+        title = user.title.toLowerCase()
+        if (title.includes('front-end')) {
+            dafaultSources.headerLogo = frontEnd;
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: change everywhere to defaultSources

File: src/cv/experience/Experience.js [hunk 3]
```diff
@@ -16,38 +15,37 @@ class Experience extends Component {
...
-                    </div>
-                    <div className="Cv-experience-experience-technologies">
-                        Technologies used: {project.technologies.reduce((a, b) => {return a + ', ' + b})}
-                    </div>
-                    </div>
-                )
+                {experience.mainProjects.map(function (project, index) {
+                    return (
+                        <div key={index}>
+                            <div className="Cv-experience-experience-desc">
```
Comment (StudentDawid) [inline src/cv/experience/Experience.js]: Why 'experience-experience' ?

File: src/cv/experience/Experience.js [hunk 4]
```diff
@@ -16,38 +15,37 @@ class Experience extends Component {
...
+                                {project.desc}
+                            </div>
+                            <div className="Cv-experience-experience-technologies">
```
Comment (StudentDawid) [inline src/cv/experience/Experience.js]: Why 'experience-experience' ?

File: src/cv/experience/Experience.js [hunk 5]
```diff
@@ -16,38 +15,37 @@ class Experience extends Component {
...
+                        <img src={circleMaroonMiddle} className="Cv-logo" alt="logo" />
+                        <div className="Cv-experience-experience-title">{project.name} as {project.title}</div>
                         <div>
-                        <div className="Cv-experience-experience-desc">
-                            {project.desc}
-                        </div>
-                        <div className="Cv-experience-experience-technologies">
-                            Technologies used: {project.technologies.reduce((a, b) => {return a + ', ' + b})}
-                        </div>
+                            <div className="Cv-experience-experience-desc">
```
Comment (StudentDawid) [inline src/cv/experience/Experience.js]: Same

File: src/cv/experience/Experience.js [hunk 6]
```diff
@@ -16,38 +15,37 @@ class Experience extends Component {
...
+                                {project.desc}
+                            </div>
+                            <div className="Cv-experience-experience-technologies">
```
Comment (StudentDawid) [inline src/cv/experience/Experience.js]: Same

File: src/cv/firstPage/firstPage.js [hunk 7]
```diff
@@ -13,79 +13,77 @@ import experience from '../../img/experience.svg';
...
+    }
 
-        if (user.experience) {
-            experiencesList = user.experience.map(function (experience, index) {
-                return (
-                    <Experience experience={experience} key={index} />
-                );
-            });
-        }
+    if (user.experience) {
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: Better way : 
const experiencesList = user.experience ? user.experience.map(function (experience, index) {...}) : [];

File: src/cv/firstPage/firstPage.js [hunk 8]
```diff
@@ -13,79 +13,77 @@ import experience from '../../img/experience.svg';
...
+    }
 
-        if (user.education) {
-            educationList = user.education.map(function (education, index) {
-                return (
-                    <Education education={education} key={index} />
-                )
-            });
-        }
+    if (user.education) {
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: Better way : 
const educationList = user. education ? user.education.map(function (education, index) {...}) : [];

File: src/cv/experience/Experience.js (hunk 4 above)
Comment (MontJamesmont) [reply src/cv/experience/Experience.js]: fixed in d5fd54753086e85194a6e17253b7bf16c6b164fd

File: src/cv/experience/Experience.js (hunk 3 above)
Comment (MontJamesmont) [reply src/cv/experience/Experience.js]: fixed in d5fd54753086e85194a6e17253b7bf16c6b164fd

File: src/cv/secondPage/secondPage.js [hunk 9]
```diff
@@ -7,62 +7,60 @@ import target from '../../img/target.svg';
...
+        profile: profile,
+        bluePartCircle: bluePartCircle
+    }
 
-        if (user.skillset) {
-            skillsetList = (
-                <Skillset skillset={user.skillset} />
-            );
-        }
+    if (user.skillset) {
```
Comment (StudentDawid) [inline src/cv/secondPage/secondPage.js]: Better way :
const skillsetList = user. skillset ? (<Skillset skillset={user.skillset} />) : [];

File: src/cv/firstPage/firstPage.js [hunk 10]
```diff
@@ -13,79 +13,77 @@ import experience from '../../img/experience.svg';
...
-            logoDS: logoDS,
-            headerLogo: headerLogo,
-            experience: experience,
-            education: education
-        }
+function FirstPage(props) {
+    let title = "";
+    let headerLogo = "";
+    let user = props.user;
+    let sources = props.sources;
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: Better way:
let { user, sources } = props;

File: src/cv/secondPage/secondPage.js [hunk 11]
```diff
@@ -7,62 +7,60 @@ import target from '../../img/target.svg';
...
-        let skillsetList = [];
-        let dafaultSources = {
-            target: target,
-            skillset: skillset,
-            profile: profile,
-            bluePartCircle: bluePartCircle
-        }
+function SecondPage(props) {
+    let user = props.user;
+    let sources = props.sources;
```
Comment (StudentDawid) [inline src/cv/secondPage/secondPage.js]: Better way:
let { user, sources } = props;

File: src/cv/firstPage/firstPage.js (hunk 1 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 4dc4a86dced005015dc9a232a5d7afb7d9c13df3

**Review by StudentDawid — CHANGES_REQUESTED** [review]

File: src/cv/firstPage/firstPage.js (hunk 2 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 4dc4a86dced005015dc9a232a5d7afb7d9c13df3

File: src/cv/experience/Experience.js (hunk 5 above)
Comment (MontJamesmont) [reply src/cv/experience/Experience.js]: fixed in d5fd547

File: src/cv/experience/Experience.js (hunk 6 above)
Comment (MontJamesmont) [reply src/cv/experience/Experience.js]: fixed in d5fd547

File: src/cv/firstPage/firstPage.js (hunk 7 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 2b33bd4cdb20dc76901776d17342a741f4d4b9f1

File: src/cv/firstPage/firstPage.js (hunk 8 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 2b33bd4

File: src/cv/secondPage/secondPage.js (hunk 9 above)
Comment (MontJamesmont) [reply src/cv/secondPage/secondPage.js]: fixed in 2b33bd4

File: src/cv/firstPage/firstPage.js (hunk 10 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 2b33bd4

File: src/cv/secondPage/secondPage.js (hunk 11 above)
Comment (MontJamesmont) [reply src/cv/secondPage/secondPage.js]: fixed in 2b33bd4

File: src/cv/firstPage/firstPage.js [hunk 12]
```diff
@@ -13,79 +13,72 @@ import experience from '../../img/experience.svg';
...
+    }
 
-        if (user.experience) {
-            experiencesList = user.experience.map(function (experience, index) {
-                return (
-                    <Experience experience={experience} key={index} />
-                );
-            });
-        }
+    experiencesList = user.experience ? user.experience.map(function (experience, index) {
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: Can be const

File: src/cv/firstPage/firstPage.js [hunk 13]
```diff
@@ -13,79 +13,72 @@ import experience from '../../img/experience.svg';
...
-                dafaultSources.headerLogo = java;
-            } else if (title.includes('test')) {
-                dafaultSources.headerLogo = test;
-            } else if (title.includes('bussiness')) {
-                dafaultSources.headerLogo = bussinessAnalyst;
-            } else if (title.includes('cq5')) {
-                dafaultSources.headerLogo = cq5;
-            }
+    if (user.title) {
+        title = user.title.toLowerCase()
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: Can be const

File: src/cv/firstPage/firstPage.js [hunk 14]
```diff
@@ -13,79 +13,72 @@ import experience from '../../img/experience.svg';
...
-        let experiencesList = [];
-        let educationList = [];
-        let dafaultSources = {
-            logoDS: logoDS,
-            headerLogo: headerLogo,
-            experience: experience,
-            education: education
-        }
+function FirstPage(props) {
+    let title = "";
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: to remove

File: src/cv/firstPage/firstPage.js [hunk 15]
```diff
@@ -13,79 +13,72 @@ import experience from '../../img/experience.svg';
...
+    let headerLogo = "";
+    let { user, sources } = props;
+    let experiencesList = [];
+    let educationList = [];
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: To remove : experiencesList, educationList

File: src/cv/firstPage/firstPage.js [hunk 16]
```diff
@@ -13,79 +13,72 @@ import experience from '../../img/experience.svg';
...
+    }) : [];
 
-        if (user.education) {
-            educationList = user.education.map(function (education, index) {
-                return (
-                    <Education education={education} key={index} />
-                )
-            });
-        }
+    educationList = user.education ? user.education.map(function (education, index) {
```
Comment (StudentDawid) [inline src/cv/firstPage/firstPage.js]: Can be const

File: src/cv/secondPage/secondPage.js (hunk 9 above)
Comment (MontJamesmont) [reply src/cv/secondPage/secondPage.js]: fixed in 3ff9fabc20e0c47f55860369499aa3653622d2f6

File: src/cv/firstPage/firstPage.js (hunk 7 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 3ff9fabc20e0c47f55860369499aa3653622d2f6

File: src/cv/firstPage/firstPage.js (hunk 8 above)
Comment (MontJamesmont) [reply src/cv/firstPage/firstPage.js]: fixed in 3ff9fabc20e0c47f55860369499aa3653622d2f6

**Review by StudentDawid — APPROVED** [review]

**Pull request merged**